src/mbgl/annotation/render_annotation_source.cpp

```diff
@@ -41,6 +41,7 @@ void RenderAnnotationSource::update(Immutable<style::Source::Impl> baseImpl_,
                        // Zoom level 16 is typically sufficient for annotations.
                        // See https://github.com/mapbox/mapbox-gl-native/issues/10197
                        { 0, 16 },
+                       {},
```

Comment:
Can we change that to `util::TileRange{}` to make this code more readable?

Reply:
That parameter represents an `optional<LatLngBounds>`. Are you suggesting to change the type of the parameter to `TilePyramid::update` ?

Reply:
No, just writing the type here to make it explicit when reading the code.